playground/vue/__tests__/vue.spec.ts

```diff
@@ -208,8 +208,11 @@ describe('hmr', () => {
 
   test('should reload when relies file changed', async () => {
     // rerender
-    editFile('Hmr.vue', (code) => code.replace('HMR', 'HMR updated'))
     await untilUpdated(() => page.textContent('h2.hmr'), 'HMR updated')
```

Comment:
`h2.hmr` already contained `HMR updated` because it was updated at
https://github.com/vitejs/vite-plugin-vue/blob/3f55553788ab11dcba930889d4f1557ddb07fbd0/playground/vue/__tests__/vue.spec.ts#L175-L176
So this `untilUpdated` asserted nothing.

That made `editFile('Hmr.vue'` and `editFile('Hmr.vue'` to run almost the same time and caused the `file-changed` message to be sent in reverse order sometimes.
When the `file-changed` message is sent in reverse order, a renderer happens and thus the test fails.